src/DropDownMenu/DropDownMenu.js

```diff
@@ -53,7 +53,7 @@ function getStyles(props, context) {
     root: {
       display: 'inline-block',
       fontSize: spacing.desktopDropDownMenuFontSize,
-      height: spacing.desktopSubheaderHeight,
+      height: spacing.desktopToolbarHeight,
```

Comment:
That's such a breaking change. Sorry, we can accept that change. We want to focus on bug fixes for the master branch with no breaking changes.